{
  "uid" : "2ab05da3df3a78f3",
  "name" : "Test submenu and button on the Press Release page",
  "fullName" : "Test Redirection of Pages: Test submenu and button on the Press Release page",
  "historyId" : "92f758c56c9af6a5eff84c5551d9dfb4",
  "time" : {
    "start" : 1698821072730,
    "stop" : 1698821086588,
    "duration" : 13858
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div/div/div/div/div[4]/div/div/div/div[1]/div[1]/div[2]/div/div/div/div[2]/h3/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55a2f1da74e3 <unknown>\n#1 0x55a2f1ad6c76 <unknown>\n#2 0x55a2f1b12c96 <unknown>\n#3 0x55a2f1b12dc1 <unknown>\n#4 0x55a2f1b4c7f4 <unknown>\n#5 0x55a2f1b3203d <unknown>\n#6 0x55a2f1b4a30e <unknown>\n#7 0x55a2f1b31de3 <unknown>\n#8 0x55a2f1b072dd <unknown>\n#9 0x55a2f1b0834e <unknown>\n#10 0x55a2f1d673e4 <unknown>\n#11 0x55a2f1d6b3d7 <unknown>\n#12 0x55a2f1d75b20 <unknown>\n#13 0x55a2f1d6c023 <unknown>\n#14 0x55a2f1d3a1aa <unknown>\n#15 0x55a2f1d906b8 <unknown>\n#16 0x55a2f1d90847 <unknown>\n#17 0x55a2f1da0243 <unknown>\n#18 0x7fae77290402 <unknown>\n\n",
  "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div/div/div/div/div[4]/div/div/div/div[1]/div[1]/div[2]/div/div/div/div[2]/h3/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55a2f1da74e3 <unknown>\n#1 0x55a2f1ad6c76 <unknown>\n#2 0x55a2f1b12c96 <unknown>\n#3 0x55a2f1b12dc1 <unknown>\n#4 0x55a2f1b4c7f4 <unknown>\n#5 0x55a2f1b3203d <unknown>\n#6 0x55a2f1b4a30e <unknown>\n#7 0x55a2f1b31de3 <unknown>\n#8 0x55a2f1b072dd <unknown>\n#9 0x55a2f1b0834e <unknown>\n#10 0x55a2f1d673e4 <unknown>\n#11 0x55a2f1d6b3d7 <unknown>\n#12 0x55a2f1d75b20 <unknown>\n#13 0x55a2f1d6c023 <unknown>\n#14 0x55a2f1d3a1aa <unknown>\n#15 0x55a2f1d906b8 <unknown>\n#16 0x55a2f1d90847 <unknown>\n#17 0x55a2f1da0243 <unknown>\n#18 0x7fae77290402 <unknown>\n\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Given The user is on the \"stg\" site",
      "time" : {
        "start" : 1698821073705,
        "stop" : 1698821079833,
        "duration" : 6128
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user hovers over the \"company\" menu",
      "time" : {
        "start" : 1698821079833,
        "stop" : 1698821080171,
        "duration" : 338
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And The user clicks on the \"press_release\" submenu link and verify the redirection \"press_release\" page",
      "time" : {
        "start" : 1698821080172,
        "stop" : 1698821081342,
        "duration" : 1170
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"telecom_certificate\" button",
      "time" : {
        "start" : 1698821081342,
        "stop" : 1698821086519,
        "duration" : 5177
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div[1]/div[3]/div/div/div/div/main/article/div/section/div/div/div/div/div[4]/div/div/div/div[1]/div[1]/div[2]/div/div/div/div[2]/h3/a\"}\n  (Session info: chrome=114.0.5735.133); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x55a2f1da74e3 <unknown>\n#1 0x55a2f1ad6c76 <unknown>\n#2 0x55a2f1b12c96 <unknown>\n#3 0x55a2f1b12dc1 <unknown>\n#4 0x55a2f1b4c7f4 <unknown>\n#5 0x55a2f1b3203d <unknown>\n#6 0x55a2f1b4a30e <unknown>\n#7 0x55a2f1b31de3 <unknown>\n#8 0x55a2f1b072dd <unknown>\n#9 0x55a2f1b0834e <unknown>\n#10 0x55a2f1d673e4 <unknown>\n#11 0x55a2f1d6b3d7 <unknown>\n#12 0x55a2f1d75b20 <unknown>\n#13 0x55a2f1d6c023 <unknown>\n#14 0x55a2f1d3a1aa <unknown>\n#15 0x55a2f1d906b8 <unknown>\n#16 0x55a2f1d90847 <unknown>\n#17 0x55a2f1da0243 <unknown>\n#18 0x7fae77290402 <unknown>\n\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.10/dist-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.10/dist-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/PageRedirectionSteps.py\", line 103, in step_impl\n    button_element = context.driver.find_element(By.XPATH, xpath)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/home/addweb/.local/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "7fa3eca8bfbe4d2b",
        "name" : "Error Message",
        "source" : "7fa3eca8bfbe4d2b.txt",
        "type" : "text/plain",
        "size" : 988
      }, {
        "uid" : "8fcf6c70e52c820d",
        "name" : "Screenshot of https://stg.voltactivedata.com/company/press-releases/ Page",
        "source" : "8fcf6c70e52c820d.png",
        "type" : "image/png",
        "size" : 115633
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then The user should be redirected to the \"telecom_certificate\" page",
      "time" : {
        "start" : 1698821086588,
        "stop" : 1698821086588,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"i2i_systems\" button",
      "time" : {
        "start" : 1698821086588,
        "stop" : 1698821086588,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then The user should be redirected to the \"i2i_systems\" page",
      "time" : {
        "start" : 1698821086588,
        "stop" : 1698821086588,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When The user clicks on the \"explosive_growth\" button",
      "time" : {
        "start" : 1698821086588,
        "stop" : 1698821086588,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then The user should be redirected to the \"explosive_growth\" page",
      "time" : {
        "start" : 1698821086588,
        "stop" : 1698821086588,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 9,
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Test Redirection of Pages"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "2ab05da3df3a78f3.json",
  "parameterValues" : [ ]
}